### PR TITLE
Update pauseMints to allowMinting

### PR DESCRIPTION
**Pull request by johnnyd-eth**
## What does this PR do and why?

- Updates fundingCycleMetadata to use allowMinting instead of pauseMints to reflect latest contract change
- Updates `yarn tests` to reflect these changes
- Minor fixes to `V2ConfirmPayModal`

## Screenshots or screen recordings

_If applicable, provide screenshots or screen recordings to demonstrate the
changes._

## Acceptance checklist

- [x] I have evaluated the [Approval Guidelines](https://github.com/jbx-protocol/juice-interface/blob/main/CONTRIBUTING.md#approval-guidelines) for this PR.
- [x] I have tested this PR in [all supported browsers](https://github.com/jbx-protocol/juice-interface/blob/main/CONTRIBUTING.md#supported-browsers).

### CODE REVIEW

File: src/components/shared/ManageTokensModal.tsx [hunk 1]
```diff
@@ -81,6 +81,8 @@ export default function ManageTokensModal({
 
   const redeemDisabled = !Boolean(hasOverflow)
 
+  console.info('Owner can mint tokens:, ', projectAllowsMint)
```
Comment (aeolianeth) [inline src/components/shared/ManageTokensModal.tsx:84]: ser

**Comment by aeolianeth** [comment]
@johnnyd-eth LGTM, but can you pls
- remove the reconfig test data and console logs
- update tests so the build passes.

Cheers ser

**Comment by johnnyd-eth** [comment]
@aeolianeth FYI, was meant to draft this. Only had it up for Contract Crew to test the reconfig tx stuff. But I will fix these and get it ready to merge now

**Comment by aeolianeth** [comment]
> @aeolianeth FYI, was meant to draft this. Only had it up for Contract Crew to test the reconfig tx stuff. But I will fix these and get it ready to merge now

my badddd

File: src/utils/v2/__tests__/fundingCycle.test.ts [hunk 2]
```diff
@@ -52,7 +52,7 @@ export function packFundingCycleMetadata(
   if (pausePay) packed = packed.or(one.shl(56))
   if (pauseDistributions) packed = packed.or(one.shl(57))
   if (pauseRedeem) packed = packed.or(one.shl(58))
-  if (pauseMint) packed = packed.or(one.shl(59))
+  if (allowMinting) packed = packed.or(one.shl(59))
```
Comment (aeolianeth) [inline src/utils/v2/__tests__/fundingCycle.test.ts:55]: @johnnyd-eth this an `pauseBurn` should be swapped

Comment (johnnyd-eth) [reply src/utils/v2/__tests__/fundingCycle.test.ts]: @aeolianeth good catch. Done

**Review by aeolianeth — APPROVED** [review]

**Pull request merged**